src/agent/LibFuzzerDotnetLoader/LibFuzzerDotnetLoader.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>10.0</LangVersion>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Requires commit ec28c46 or greater. -->
+    <ProjectReference Include="../sharpfuzz/src/SharpFuzz/SharpFuzz.csproj" />
```

Comment:
This is gross, and rests atop load-bearing CI scripts. There may be a better way, but this is at least blunt. Regardless, we can change that after https://github.com/Metalnem/sharpfuzz/issues/26 is resolved.